subsys/bluetooth/mesh/health_srv.c

```diff
@@ -198,12 +212,10 @@ static void health_fault_test(struct bt_mesh_model *model,
 	BT_DBG("test 0x%02x company 0x%04x", test_id, company_id);
 
 	if (srv->cb && srv->cb->fault_test) {
-		int err;
```

Comment:
Why are you moving this from branch scope to function scope? As far as I can see it's not used anywhere outside of the branch.

Reply:
I needed that variable in function scope when I was also processing an error from bt_mesh_model_send() but then I reverted that change and forgot to move the variable back to branch scope. I fixed it now.